### PR TITLE
add melting point dataset

### LINKED ISSUE #417: add melting point dataset

### PR DESCRIPTION
**Pull request by kjappelbaum**
Fixes #417

### CODE REVIEW

File: data/tabular/melting_points/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
+---
+name: perovskite_db
+description: |-
+    Literature mined data on melting points of organic compounds.
+targets:
+    - id: mp
+      description: mean melting point
+      units: deg C
+      type: float
```
Comment (MicPie) [inline data/tabular/melting_points/meta.yaml:9]: `continous`

**Review by MicPie — CHANGES_REQUESTED** [review]
lgtm when `float` is also changed to `continous`. 👍

File: data/tabular/melting_points/meta.yaml (hunk 1 above)
Comment (kjappelbaum) [inline data/tabular/melting_points/meta.yaml:9]:
```suggestion
      type: continous
```

File: data/tabular/melting_points/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      names:
+          - noun: mean melting point
+      uris:
+    - id: mp_range
+      description: melting point range
+      units: deg C
+      type: float
```
Comment (kjappelbaum) [inline data/tabular/melting_points/meta.yaml:16]:
```suggestion
      type: continous
```

**Comment by kjappelbaum** [comment]
made you changes, @MicPie and rebased on `main`.

**Review by MicPie — APPROVED** [review]
lgtm

**Pull request merged**